docs/development.md

```diff
@@ -95,6 +95,7 @@ sudo apt install clang ccache lld
 
 #### Configure for Building
 
+1. **If you haven't already**, [activate the Python environment](#set-up-the-python-environment)
```

Comment:
```suggestion
1. *If you haven't already*, [activate the Python environment](#set-up-the-python-environment)
```
maybe italic instead of bold?

Reply:
Well it looks like a very strong and not too kind invitation but I don't have too strong feelings about this. Probably depends on prior experience and I am rather used to _always_ create and use a venv thus the reminder could be more gentle without shouting at me ;) But as mentioned, no super strong feelings.